spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -498,6 +498,7 @@
       expect(page).not_to have_content(t('doc_auth.tips.document_capture_selfie_text1'))
       attach_images
       continue_doc_auth_form
+      expect(page).to have_title(t('doc_auth.headings.selfie_capture'))
```

Comment:
@AShukla-GSA  ... this could be an opportunity to add:

expect(page).to have_title(t('doc_auth.headings.document_capture')) # above line 499